ReactWindows/ReactNative/Views/TextInput/ReactTextInputManager.cs

```diff
@@ -753,5 +755,21 @@ private void OnKeyDown(object sender, KeyRoutedEventArgs e)
                             e.Key));
             }
         }
+
+        private void OnKeyUp(object sender, KeyRoutedEventArgs e)
+        {
+            var textBox = (ReactTextBox)sender;
+            if (!e.Handled)
```

Comment:
Why do we need to check for Handled here? I believe the routed event infrastructure would prevent this event from being called if it is already handled.

Reply:
We use a `Handled` check in the KeyDown callback because it is set within the event handler.